.travis.yml

```diff
@@ -11,9 +11,9 @@ before_install:
   # PRs to master are only ok if coming from dev branch
   - '[ $TRAVIS_PULL_REQUEST = "false" ] || [ $TRAVIS_BRANCH != "master" ] || ([ $TRAVIS_PULL_REQUEST_SLUG = $TRAVIS_REPO_SLUG ] && [ $TRAVIS_PULL_REQUEST_BRANCH = "dev" ])'
   # Pull the docker image first so the test doesn't wait for this
-  - docker pull nfcore/rnaseq
+  - docker pull nfcore/rnaseq:dev
```

Comment:
Hm, should this not be something else? 

Like stay `latest` or specify a `tag` ?

Reply:
...don't think so? This is what we did in the tools template. Idea being that the `dev` branch should always have the most up to date code and so docker build for testing.

Reply:
True, you're right...